js/app.js

```diff
@@ -0,0 +1,212 @@
+// 1. boolean
+// 2. string
+// 3. Objects
+// 4. Array
+// 5. Objects - with strings
+// 6. Objects - with a strings and an array
+
+		// Take it easy
+
+// 1. 
+
+let rainbowColors = ["red", "orange", "yellow", 'green', 'blue', 'indigo', 'violet'];
+
+// // 2. 
+
+console.log(rainbowColors[4])
+
+// // 3. 
+
+let anthony = {
+	favFood: "tacos",
+	hobbies: ["biking"],
+	currentHometown: "Chicago, IL",
+	favDatatype: "boolean"
+}
```

Comment:
what can i say the kids love booleans.  anyway, i won't ding you because i think it's clever that you allowed for more hobbies in the future, but the instructions technically said "a hobby."